Basic Web Server Setup(Ubuntu)/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+echo "Hello There!!!"
+echo "Excited to set up your new server! "
+
+echo "Updating packages!"
+sudo apt update
+
+echo "Installing nginx!"
+sudo apt install nginx -y
+
+echo "Adjusting your firewall rules! \n So that no one can get into your system easily"
+sudo apt-get install ufw
+sudo ufw enable -y
+sudo ufw app list
+sudo ufw allow 'Nginx HTTP'
+sudo ufw status
+
+echo "Checking your web server"
+sudo systemctl status nginx
```

Comment:
holds terminal from executing the script any further, and awaits user to press `q`